config_mcmc.yml

```diff
@@ -0,0 +1,249 @@
+#        spread: 
+#          # Spread for initializing walkers
+#          - [1.0E-8, 1.0E-4, .NAN, .NAN]
+#          - [1.0E-8, 1.0E-4, .NAN, .NAN]
+#          - [1.0E-8, 1.0E-4, .NAN, .NAN]
+#          - [1.0E-8, 1.0E-4, .NAN, .NAN]
+#          - [1.0E-8, 1.0E-4, .NAN, .NAN]
+#          - [1.0E-8, 1.0E-4, .NAN, .NAN]
+    contrast:  # raw contrast
+      prior_ftn:
+        - .NAN
+        - .NAN
+        - uniform
+        - .NAN
+        - .NAN
+        - .NAN
+      prior_args:
+        lower_bound:
+          - .NAN
+          - .NAN
+          - 1.0e-10
+          - .NAN
+          - .NAN
+          - .NAN
+        upper_bound:
+          - .NAN
+          - .NAN
+          - 1.0e-9
+          - .NAN
+          - .NAN
+          - .NAN
+      ini_pars:
+        center:
+          - .NAN
+          - .NAN
+          - 1.500000000000000311e-10
+          - .NAN
+          - .NAN
+          - .NAN
+        spread:
+          - .NAN
+          - .NAN
+          - 2.000000000000000311e-12
+          - .NAN
+          - .NAN
+          - .NAN
+#    throughput:  # core throughput
+#      prior_ftn:
+#        - .NAN
+#        - .NAN 
+#        - uniform
+#        - uniform
+#        - uniform
+#        - .NAN 
+#      prior_args:
+#        lower_bound:
+#          - .NAN
+#          - .NAN
+#          - 0.03
+#          - 0.03
+#          - 0.03
+#          - .NAN
+#        upper_bound:
+#          - .NAN
+#          - .NAN
+#          - 0.3
+#          - 0.3
+#          - 0.3
+#          - .NAN
+#      ini_pars:
+#        center:
+#          - .NAN
+#          - .NAN
+#          - 0.15
+#          - 0.15
+#          - 0.15
+#          - .NAN
+#        spread:
+#          - .NAN
+#          - .NAN
+#          - 0.0015
+#          - 0.0015
+#          - 0.0015
+#          - .NAN
+      
+# Initial argument values for ebs.ErrorBudget.run_etc()
+
+paths:
+  output: ./outputs
+  input: ./inputs
+
+input_files:
+  contrast: mcmc_contrast.csv
+  throughput: mcmc_throughput.csv
+  wfe: mcmc_wfe.csv
+  wfsc_factor: mcmc_wfsc.csv
+  sensitivity: mcmc_sensitivity.csv
+
+#json_files:
+#  ref_json: mcmc_ref.json  # An input specifiying the 'unaberrated' observatory
+#  pp_json: mcmc_pp.json  # An intermediate file read in by EXOSIMS
+
+targets:
+#  star1:
+#    HIP: 32439 A
+#    spec_type: F8V
+#    luminosity: 0.2615
+#    eeid: 0.07423
+#    eepsr: 6.34e-11
+#    exo_zodi: 3.0
+#  star2:
+#    HIP: 77052 A
+#    spec_type: G5V
+#    luminosity: -0.0788
+#    eeid: 0.06174
+#    eepsr: 1.39e-10
+#    exo_zodi: 3.0
+  star3:
+    HIP: 79672 
+    spec_type: G2Va
+    luminosity: 0.0391
+    eeid: 0.07399
+    eepsr: 1.06e-10
+    exo_zodi: 3.0
+#  star4:
+#    HIP: 26779 
+#    spec_type: K1V
+#    luminosity: -0.3209
+#    eeid: 0.05633
+#    eepsr: 2.42e-10
+#    exo_zodi: 3.0
+#  star5:
+#    HIP: 113283
+#    spec_type: K4Ve
+#    luminosity: -0.70
+#    eeid: 0.05829
+#    eepsr: 5.89e-10
+#    exo_zodi: 3.0
+    
+
+initial_exosims: 
```

Comment:
In this case here, I would just specify the `mcmc` parameters in the `.yml` file and read in the exosims parameters from the `ref.json` file to make it consistent with the non-mcmc case.